datafusion/sqllogictest/test_files/case.slt

```diff
@@ -61,7 +61,7 @@ NULL
 6
 NULL
 NULL
-7
```

Comment:
sam thing here, `NULL > 2` is not `true` (it isn't false either, it is NULL) so this should not take the value of `b`